apps/emqx_gateway/src/coap/emqx_coap_api.erl

```diff
@@ -83,7 +83,7 @@ request_parameters() ->
 request_properties() ->
     properties([ {token, string, "message token, can be empty"}
                , {method, string, "request method type", ["get", "put", "post", "delete"]}
-               , {timeout, string, "timespan for response", "10s"}
+               , {timeout, string, "timespan for response"}
```

Comment:
the fourth parameter(10s) is an enum, not an example prop. so remove this
warning by swagger edit: enum must be an array